tests/test_e2ee_unified.py

```
@@ -44,6 +49,13 @@
     IMPORTS_AVAILABLE = False
 
 
+def _make_capabilities(**overrides) -> MatrixLibraryCapabilities:
```

Ruff (0.15.12)

[warning] Missing type annotation for `**overrides`

(ANN003)